binaries/build_wheels.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A script that downloads ffmpeg, ffprobe, and packager static builds for all
+the platforms we build for and then builds distribution wheels for them.
+"""
+
+import os
+import sys
+import urllib.request
+import setuptools # type: ignore
+import setuptools.command.build_py #type: ignore
+import streamer_binaries
+
+
+# Version constants.
+# Change to download different versions.
+FFMPEG_VERSION = 'n4.4-1'
+PACKAGER_VERSION = 'v2.99.5'
+
+# A map of postfixes that will be combined with the binary download links
+# to achieve a full download link.  Different postfix for each platform.
+# Extend this dictionary to add more platforms.
+PLATFORM_POSTFIXES = {
```

Comment:
Perhaps rename to "suffix" instead of "postfix".  I checked myself on Wikipedia, since perhaps I'm just not used to hearing postfix for the ending of a word.  What I found was the [disambiguation article](https://en.wikipedia.org/wiki/Postfix) that says:

> Postfix (linguistics), an affix which is placed after the stem of a word

But when I click [the link](https://en.wikipedia.org/wiki/Postfix_(linguistics)), it redirects to [the article for "suffix".](https://en.wikipedia.org/wiki/Suffix)

Language is hard and dumb and arbitrary, and English more than most, and **_American_** English...